public/styles/main.css

```diff
@@ -450,23 +459,34 @@ main {
   margin-bottom: 0rem;
 }
 
+.query-partial-container {
+  padding: 5px;
+}
+
 .container-scope3-carbon {
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
+  overflow: hidden;
   min-height: 30em;
 }
 
-.box1 {
-  flex: 0 0 40%;
+.scope3-co2-left {
```

Comment:
if we are renaming `box1` and `box2` should `box3` (below) also be renamed in accordance with this?

Reply:
well, really I think we should have two elements here `.row` and `.col` either flex/grid which should allow us to shape/lay out things the way we want